src/org/linphone/ContactsManager.java

```diff
@@ -565,7 +563,7 @@ public void run() {
 							continue;
 
 						//Remove linphone contact tag if the contact has no sip address
-						if (LinphoneActivity.isInstanciated() && LinphoneActivity.instance().getResources().getBoolean(R.bool.use_linphone_tag)) {
```

Comment:
Sanity check. This boolean will always be true and is tied to linphone implementation